sdk/keyvault/azure-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/KeyClient.java

```diff
@@ -755,11 +768,14 @@ public Key restoreKey(byte[] backup) {
      * <p>Lists the deleted keys in the key vault and for each deleted key prints out its recovery id.</p>
      * {@codesnippet com.azure.keyvault.keys.keyclient.listDeletedKeys#Context}
      *
+     * <p>Iterate over the lists the deleted keys by each page in the key vault and for each deleted key prints out its recovery id.</p>
```

Comment:
Description is fine, But the header should stay too.
`<p><strong>Code Samples to iterate over Deleted Keys by page</strong></p>`